### ISSUE TITLE
Publish new version?

### ISSUE THREAD

**Issue by priley86** [lerna/lerna-changelog#122]
Hi, 

I'm noting that the `nextVersion` option with lerna-changelog `0.8.0` does not currently seem to respect my passed option in `lerna.json`. However, if I clone the source of this project, build it, and use that instead, it seems to work and replace the title just fine...

Do you plan to release this repo again soon?

Here is my repo for reference...
https://github.com/patternfly/patternfly-react/blob/master/lerna.json#L12

CHANGELOG.md title is still "Unreleased":
https://github.com/patternfly/patternfly-react/blob/master/CHANGELOG.md#unreleased-2018-10-08

**Comment by Turbo87** [comment]
> Do you plan to release this repo again soon?

I tried to do this just now and have pushed a v0.8.1 tag to the repo, but unfortunately npm isn't allowing me to publish right now due to:
```
npm ERR! publish Failed PUT 401
npm ERR! code E401
npm ERR! This package requires that publishers enable TFA and provide an OTP to publish. For more info, visit: https://go.npm.me/2fa-guide : lerna-changelog
```

Enabling TFA and OTP is somewhat problematic for me though since I use my npm account for publishing via TravisCI and AFAICT that does not support TFA/OTP.

tl;dr If I would want to publish `lerna-changelog` I would have to break publishing of a lot of my other projects. Not sure how to resolve this... 😕 

/cc @hzoo

**Comment by hzoo** [comment]
I can turn it off if you want, and I can publish now

**Comment by Turbo87** [comment]
@hzoo that would be great, thanks! I'll see if I can find a better long-term solution.

**Comment by hzoo** [comment]
Oh I already published btw, 0.8.1

**Comment by Turbo87** [comment]
Awesome, thanks again!
